exercises/practice/parallel-letter-frequency/.meta/example.rb

```diff
@@ -0,0 +1,26 @@
+class ParallelLetterFrequency
+  def self.count(texts)
+    rs = (0...texts.length).map do |i|
+      Ractor.new(texts[i]) do |text|
+        res = Hash.new(0)
+        text.downcase.each_grapheme_cluster do |c|
+          res[c] += 1 if c.match?(/\p{Alpha}/)
+        end
+
+        res
+      end
```

Comment:
What words do `rs` and `res` and c` represent?

It looks like `res` might be a tally, while `c` might be `cluster`, or `grapheme_cluster` while I think `cluster` is probably informative enough.

```ruby
tally[cluster] += 1 if cluster.match?(/\p{Alpha}/)
```

This requires less deciphering in reading, in my opinion, more understanding built in  to the reading.

Reply:
Clarified variable names.